scripts/post-build.js

```diff
@@ -32,20 +32,20 @@ module.exports = function(fs, path, log, options, done) {
 
       // cmi5 is a profile of the xAPI specification and some systems, e.g. SCORM Cloud
       // do not work well with both manifest types, so remove the one which is not being used.
-      if (!configJson._xapi.hasOwnProperty('_specification') || configJson._xapi._specification === 'xAPI') {
-        // TODO - This will change once cmi5 is supported.
-        // Remove the cmi5.xml file (default behaviour).
-        // fs.unlink(path.join(options.outputdir, 'cmi5.xml'), callback);
-        callback();
-      } else if (configJson._xapi._specification === 'cmi5') {
-        // Remove the tincan.xml file.
-        fs.unlink(path.join(options.outputdir, 'tincan.xml'), callback);
+
+        if (configJson?._xapi?._specification === 'xAPI') {
+          // Remove the cmi5.xml file (default behaviour).
+          fs.unlink(path.join(options.outputdir, 'cmi5.xml'), callback);
+        } else if (configJson._xapi._specification === 'cmi5') {
+          // Remove the tincan.xml file.
+          fs.unlink(path.join(options.outputdir, 'tincan.xml'), callback);
+        }
```

Comment:
Because of the original empty setting check, would the new logic be safer as the following?
```suggestion
        if (configJson?._xapi?._specification !== 'cmi5') {
          // Remove the cmi5.xml file (default behaviour).
          fs.unlink(path.join(options.outputdir, 'cmi5.xml'), callback);
        } else {
          // Remove the tincan.xml file.
          fs.unlink(path.join(options.outputdir, 'tincan.xml'), callback);
        }
```